conanfile.py

```diff
@@ -2,12 +2,12 @@
 
 
 class Conanfile(ConanFile):
-    name = "Boids"
+    name = "RealTimeParticles"
     version = "0.1"
     requires = ["sdl/[>=2.0.12]",
                 "glad/[>=0.1.29]",
-                "opencl-headers/[>=2021.04.29]",
-                "opencl-icd-loader/[>=2021.04.29]",
+                #"opencl-headers/[>=2021.04.29]",
```

Comment:
Is it on purpose or a left over? Pretty sure it is not going to work on a clean Windows machine if we don't get those two.

Reply:
reverted ;)